index.html

```diff
@@ -83,240 +91,244 @@ <h4>LET'S CONNECT</h4>
-            <ul>
               <li>
-                <a href="#">
-                  <img src="img/github.png" alt="github picture" />
-                </a>
+                <a href="#"
+                  ><img src="./img/angellist.png" alt="angellist picture"
+                /></a>
               </li>
             </ul>
-
-            <ul>
+            <button type="button">get my resume</button>
+          </div>
+          <div class="tooling">
+            <h3>
+              <span>Languages</span><img src="./img/Frame-d.svg" alt="" />
+            </h3>
+            <ul class="stack">
               <li>
-                <a href="#">
-                  <img src="img/angellist.png" alt="angellist picture" />
-                </a>
+                <img src="./img/Frame-js.svg" alt="" />&nbsp;&nbsp;&nbsp;<span
+                  >JavaScript</span
+                >
+              </li>
+              <li>
+                <img src="./img/Frame-html.svg" alt="" />&nbsp;&nbsp;&nbsp;<span
+                  >HTML</span
+                >
+              </li>
+              <li>
+                <img src="./img/Frame-css.svg" alt="" />&nbsp;&nbsp;&nbsp;<span
+                  >CSS</span
+                >
               </li>
             </ul>
+            <h3>
+              <span>Frameworks</span><img src="./img/Frame-r.svg" alt="" />
+            </h3>
+            <h3 class="cnt">
+              <span>Skills</span><img src="./img/Frame-r.svg" alt="" />
+            </h3>
           </div>
-
-          <button type="button">Get my resume</button>
-
-          <ul>
-            <li>
-              <div class="languages-skills">
-                <img src="./img/Frame7.svg" alt="" />
-              </div>
-            </li>
-            <li>
-              <div class="my-skills">
-                <img src="./img/Frame9(1).svg" alt="" />
-                <img src="./img/Frame10.svg" alt="" />
-                <img src="./img/Frame11.svg" alt="" />
-              </div>
-            </li>
-            <li>
-              <div class="languages-skills">
-                <img src="./img/Frame8.svg" alt="" />
-              </div>
-            </li>
-            <li>
-              <div class="languages-skills">
-                <img src="./img/Frame9.svg" alt="" />
-              </div>
-            </li>
-          </ul>
-        </section>
+        </div>
       </section>
 
-      <section class="contact-me">
-        <br />
-        <h6>Contact me</h6>
-        <p>
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I'd love to help
-          with it.
-        </p>
+      <section id="contact" class="contact">
+        <div class="contact-container">
+          <h6>Contact me</h6>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I'd love

```

Comment:
- [ ] The submit button should be center aligned as in the Figma  design for the desktop version while it is left aligned in the mobile version.
### Figma
![contact-button-center-figma](https://user-images.githubusercontent.com/44057863/168441505-160539f5-6f59-4952-a876-c26070c0f5e6.png)
### Imp
![contact-button-center-impl](https://user-images.githubusercontent.com/44057863/168441513-f8c74c24-62d4-4f55-b983-b00eb76b3926.png)
l